dashboard/app/models/levels/level.rb

```diff
@@ -448,6 +448,7 @@ def concept_level?
   # Returns an array of all the contained levels
   # (based on the contained_level_names property)
   def contained_levels
+    return [] unless is_a? Blockly
```

Comment:
this seems redundant with checking for `"contained_level_names"` below; am I misunderstanding something?

Reply:
Good question, it took me some time to figure out the answer to this! :-) The check below raises when you try to look at `properties["contained_level_names"]` non-Blockly levels, which don't have this property in their serialized attributes. I'm open to guarding against this a different way if you have one to recommend.

Reply:
Ah, in that case I'd rather update the check below to accomodate that case. Given that `contained_level_names` should be defined as a serializedproperty, we also probably don't need to reference the properties hash directly. Maybe something like

```ruby
def contained_levels
     names = try(:contained_level_names)
     return [] unless names.present?
     names.map do |contained_level_name|
```

Reply:
Thanks Elijah! That looks better.